resources/js/components/users/index.vue

```diff
@@ -12,53 +12,68 @@
 
     <p class="mt-3">Current Page: {{ currentPage }}</p>
 
-    <b-table
-        id="my-table"
-        :items="listUsers"
-        :per-page="perPage"
-        :current-page="currentPage"
-        medium
-    ></b-table>
+    <table class="table">
+      <thead>
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Name</th>
+          <th scope="col">Email</th>
+          <th scope="col">Created at</th>
+          <th scope="col">Updated at</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="item in listPaginatedUsers">
+          <th scope="row">{{ item.id }}</th>
+          <td>{{ item.name }}</td>
+          <td>{{ item.email }}</td>
+          <td>{{ item.created_at }}</td>
+          <td>{{ item.updated_at }}</td>
+        </tr>
+      </tbody>
+    </table>
+
   </div>
 </template>
 
 <script>
-  export default {
-    data() {
-      return {
-        perPage: 3,
-        currentPage: 1,
-        listUsers: listUsers
-      }
-    },
-    computed: {
-      rows() {
-        return this.listUsers.length
-      }
-    },
-    methods: {
-        loadPagination (selectedPageNumber) {
-            if (selectedPageNumber == this.currentPage) {
-                console.log('no load')
-            } else {
-                axios.get('/users/paginate/' , {
-                        params: {
-                          page: selectedPageNumber
-                        }
-                    })
-                  .then(function (response) {
-                    // handle success
-                    console.log(response);
-                  })
-                  .catch(function (error) {
-                    // handle error
-                    console.log(error);
-                  })
-                  .finally(function () {
-                    // always executed
-                  });
+    import Common from '../../services/common';
+
+    export default {
+        data() {
+            return {
+                perPage: perPage,
+                currentPage: currentPage,
+                listPaginatedUsers: listPaginatedUsers,
+                total: total,
+                paginateUrl: laroute.route('users.index'),
+            }
+        },
+        computed: {
+            rows() {
+                return this.listPaginatedUsers.length
+            }
+        },
+        methods: {
+            loadPagination (selectedPageNumber) {
+                if (selectedPageNumber == this.currentPage) {
```

Comment:
ko nên if/else ntn. dùng 1 if thôi.
if (selectedPageNumber != this.currentPage)